### ISSUE TITLE
Project

### ISSUE THREAD

**Issue by CreaturePhil** [FakeSloth/todo#1]
We should work on a project together! Got any ideas?

If you know anybody else that wants to join in, invite them in the organization.
Also if you don't like the name, suggest some alternatives for a better organization name.

@gigavinyl @stevoduhhero @Kouhai-Senpai

**Comment by margaretjoanmiller** [comment]
FakeSloth is a fantastic name but we should work on a minimalist logo; but that's not a priority. As far as new projects go, well, since I have change languages for that robotics club we probably shouldn't start anything super big just yet. However, what's your opinion on making contributions to open source projects here on GitHub in the meantime?

**Comment by CreaturePhil** [comment]
Yeah, we should probably work on small projects together first. Also you can work with multiple languages at the same time :laughing: . You can contribute to open source projects. Just pick one you want to work on, clone it and make it working locally, and start hacking away on a new feature to fixing a bug.

**Comment by CreaturePhil** [comment]
@kouhai-senpai @stevoduhhero Reminder to join organization :grinning:

**Comment by CreaturePhil** [comment]
@gigavinyl @Kouhai-Senpai @stevoduhhero @n19
https://github.com/FakeSloth/til which is inspired by https://github.com/thoughtbot/til. Like the idea?

**Comment by margaretjoanmiller** [comment]
Yeah, that's pretty cool! :)

**Comment by CreaturePhil** [comment]
@gigavinyl Start contributing now then! :+1: Also can you merge my pull request?

**Comment by CreaturePhil** [comment]
@Kouhai-Senpai @sevagsatamian Got any ideas yet?

**Comment by CreaturePhil** [comment]
@Kouhai-Senpai @sevagsatamian @gigavinyl @FakeSloth/owners @stevoduhhero

What about a reddit like forum that we create one in each language:
JavaScript and Ruby

I would do other languages like Haskell but no one in the organization has proficiency in it including me. :laughing: Each different language of the forum gets a cool name but we will have an overarching name that means all of them. @gigavinyl We can do C++ too so you can get more practice with it, since C++ is not really good at web development, we'll probably use it for some database or backend functionality. So what do you guys think of the idea? Good? Bad? Suggestions? Feedback? Got cool names?

**Comment by CreaturePhil** [comment]
Possible names:
buma
spurtflare
oppression
zie
hyp3d
~~strife~~
inferno
wulu
impossible
epicness
icelance
redddit :laughing: 

I choose these names cause they are currently available on heroku.

**Comment by ghost** [comment]
I know html and CSS we could add that to 

-Sevag-

**Comment by ghost** [comment]
Buma 

-Sevag-

**Comment by CreaturePhil** [comment]
@sevagsatamian Can you do [jade](http://jade-lang.com)?

**Comment by GreysonCode** [comment]
Icelance gets my vote.

**Comment by ghost** [comment]
I don't know jade 

-Sevag-

**Comment by CreaturePhil** [comment]
@sevagsatamian Learn it!

**Comment by CreaturePhil** [comment]
@gigavinyl Can we get your vote?

**Comment by margaretjoanmiller** [comment]
@CreaturePhil wulu

**Comment by CreaturePhil** [comment]
Damn three way split! :angry: I like one word name better so I made a strawpoll between buma vs. wulu. http://strawpoll.me/3759489

@FakeSloth/owners VOTE NOW!

**Comment by CreaturePhil** [comment]
@FakeSloth/owners Sorry I haven't been super active on this organization as I should have because I am the leader lol. I just been busy with school and have tons of other projects I'm working on to improve my Computer Science knowledge and programming abilities. In the meanwhile, some of the members are going to LAHacks http://lahacks.com. So we should have a chance to recruit more members to this organization!

**Comment by CreaturePhil** [comment]
@gigavinyl @Kouhai-Senpai @stevoduhhero
Voting ends in one day! Hurry up and get your vote in! http://strawpoll.me/3759489

**Comment by GreysonCode** [comment]
My biggest question at the moment is what the project will be. A name is nice and all, but I'm not seeing a general consensus on what we'll actually be working on. 
Some ideas to consider:
- What audience do we want to attract?
- What type of project are we comfortable with working on?
- What languages do we want to focus on?

I know that @CreaturePhil is really into Angular.js as well as Haskell, but I know neither of us know a lot about them. Same as @stevoduhhero. We're pretty comfortable with JavaScript, C++, Java, HTML, and CSS from what I know, but what languages is everyone here most comfortable with?

**Comment by CreaturePhil** [comment]
I was going to start a rough prototype of what we are going to be working on when we first decided on the name. Once we choose the name, we can move all the discussion on that repository.
- **What audience do we want to attract?** Anybody really. This app is just to learn and see how our team collaboration goes on a real project. Our best way to spread our audience is word of mouth.
- **What type of project are we comfortable with working on?** We don't know yet so we have to try out a few projects to see. We could discuss about it but who knows what we are comfortable with? It seems so far that most members are comfortable with web development.
- **What languages do we want to focus on?** Probably JavaScript since mostly everybody in the org is decent at it. We could also do Ruby or C++ depending on @gigavinyl wants practice on. For me, I can do any programming language. It really doesn't matter.

I probably will not be using Haskell in any projects in this org yet. Angular.js is not difficult to learned and I can recommend many resources to learn it in about a day or two. You don't need it yet though so when you feel like you want to learn it then you should.

I think we should all try at least one different programming language from what we are comfortable with because it can expand our programming horizons and further our abilities to code. We should all focus on the same one so we can share in the struggles together. I nominate either Ruby, Python, or C! I think C++ is too monolithic of a language and it takes a huge amount of time compare to other languages to write idiomatic, clean, modular, and clear code with C++.

In addition, no matter what project or programming language, we are going to have to write a lot of documentation! :wink:

**Comment by GreysonCode** [comment]
I'd be in for Python. It's an incredibly useful language, and I've been wanting to learn it for a long time.

**Comment by GreysonCode** [comment]
And it seems that Buma has won the name for the Project. All hail Project Buma!

**Comment by CreaturePhil** [comment]
We can use wulu for something else I have planned. :)

**Comment by CreaturePhil** [comment]
[buma](https://github.com/FakeSloth/buma/issues) has been created! Check out the issues to get the project rolling!

**Comment by CreaturePhil** [comment]
@FakeSloth/owners I got a project called `Polygot` where we try new programming languages each week. We do this by implementing some program (I don't know what program, we go to decide! Cannot be hello world! Maybe something **FakeSloth** related idk...) in that new programming language. We create a README.md explaining it and how to set up the development environment for that programming language like what to install and stuff. What do you guys think? And what program should we make with each one? What language should we start first, easy or hard? :grimacing:

**Comment by CreaturePhil** [comment]
Also, we should each do our own program in the programming language. We can work on getting setup for that language first. I vote we start with `C++` or `Python` first. :) Start easy.

**Comment by CreaturePhil** [comment]
Project suggestions:
- Project Euler problem
- Todo List

**Comment by margaretjoanmiller** [comment]
Sorry I haven't spoken here recently; homework has been hell this past week. I like the votes for Python as, like what @CreaturePhil said, C++ is not really something you can learn quickly like is the idea for `Polyglot`. Also, to clarify, is the idea that we all create one program each for each language?

**Comment by CreaturePhil** [comment]
We each create our own program for each language. We can also work on one together as well.

**Comment by CreaturePhil** [comment]
Wow, this issue is getting kind of long. Hopefully, we can get **buma** release so we can move "project discussion" other there. I think we are kind of spread thin at the moment. Most of us are busy and we need more people to help make this organization more active. We should make a recruitment program to get some more people xD :laughing:

**Comment by margaretjoanmiller** [comment]
I might not be able to go to LA Hacks but we could recruit some people there.

**Comment by CreaturePhil** [comment]
So is anybody still alive here? I'm still willing to do projects. :stuck_out_tongue_closed_eyes:

**Comment by margaretjoanmiller** [comment]
@CreaturePhil While I don't really have any ideas now, do you want to move the chat from this empty github repo to something like [slack](https://slack.com)? I've already sent invitations to the team members with their email addresses listed on their github profiles.

**Comment by CreaturePhil** [comment]
@gigavinyl I had **fakesloth**  slack domain made but forgot the password to it. I'll have to get it setup so we can use that instead of **fakeslothteam** lol

**Comment by margaretjoanmiller** [comment]
@CreaturePhil I was wondering who took [fakesloth.slack.com](https://fakesloth.slack.com) :smile:. Just tell us when it's ready.

**Comment by CreaturePhil** [comment]
Since we don't have any original ideas, why not try to make an existing app better? We can fork the codebase and improve or better yet create it from scratch allowing for my flexbility for our creative inputs to making this clone unique. By far, my most used application is Google Chrome, which a web browser is far beyond my ability to make (well a good one that is, I did make a shitty java one like a year ago [it was not very good, like it was pretty bad lol]). Social media sites are cool but unless we get users, we won't feel motivated to use them or keep improving it, in addition there is already a ton out there. We can create a game but we would need an artist to create a good looking one. Check out http://challengepost.com for some ideas.
